contents/founders/how-come-we-ship-so-much.md

```diff
@@ -0,0 +1,136 @@
+* Have complete access to metrics like revenue, hosting costs, and (soon) staff costs, for their product.
+
+The concept internally is a "[product engineer](/blog/what-is-a-product-engineer)" – i.e. an engineer who is product-minded and autonomous, rather than mindlessly filing tickets.
+
+We have one product manager who steps in reactively if a small team needs more support.
+
+The above requires a lot of context setting from the company. To that end, we're *unusually transparent*. We share board slides, financial information, fundraising progress before, during and after it happens, and much more.
+
+### 3. Hiring with intention
+
+I think 90%+ of an organization's problems are solved by hiring right – see [what we've learned about hiring](https://newsletter.posthog.com/p/everything-weve-learned-about-hiring) in our Substack newsletter.
+
+We've found it invaluable to pay people during our hiring process to do some actual work. Over time, we've refined this down to a one-day task, which gives us a good sense of how much someone can get done and the quality of their actual code.
+
+We tend to hire people with more experience, who we find work better with more autonomy. I think this is the magic combo when you're an all remote company, though we're working on ways to get better at growing people who are earlier in their careers.
+
+### 4. Each team is a mini startup
+
+I went to a superb talk by [Jeff Lawson](https://www.linkedin.com/in/jeffiel), who runs [Twilio](https://twilio.com/). He described how the greatest innovation at [Amazon](https://www.amazon.com/) was that it felt like thousands of small startups. Simply speaking, a startup gets more done per person than a big corporate.
+
+PostHog works like a group of startups with lots of [small teams](/handbook/company/small-teams). Each of our products has a team of up to 6 who can ship with minimal interference from the rest of the company. We hire a lot of former technical founders to help here.
+
+### 5. Nearly everyone is an engineer
+
+It's simple, but if you have lots of spending and focus outside of engineering, you won't get as much built.
+
+At the time of writing, [we are 37 people](/team), only 11 of whom are in Go To Market or Operations roles (and 6 of those 11 have at least some engineering in their background).
+
+We don't do outbound sales, our [marketing team](/handbook/small-teams/marketing) is very small, we only have one product manager, and our [exec team](/handbook/small-teams/exec) is only 3 people. These things are all by design. 
+
+We're product-led because our ideal users, engineers, want to try something rather than sit on a call.
+
+### 6. No meeting Tuesdays and Thursdays
+
+We want a culture where people can deep work. We push back on unnecessary meetings. 
+
+We have learned, often painfully, that some meetings are necessary, but we default to asynchronous communication and adapt where needed.
+
```

Comment:
Another fun, related example is that the engineering sprint meeting was once 45 minutes but it usually took 25 minutes, so Tim just changed it to 25 minutes.

Reply:
Added a line about this.